packages/web-app-files/src/index.js

```diff
@@ -72,6 +73,16 @@ const navItems = [
       path: `/${appInfo.id}/shares/via-link`
     }
   },
+  {
+    name: $gettext('Spaces'),
+    iconMaterial: 'space',
+    route: {
+      path: `/${appInfo.id}/spaces/projects`
+    },
+    enabled(capabilities) {
+      return capabilities.spaces && capabilities.spaces.enabled === true
```

Comment:
Can be shortened to `capabilities.spaces?.enabled`

Reply:
Unfortunately not: `SyntaxError: Unexpected token (85:54) in /Users/jannik/projects/owncloud/web/packages/web-app-files/src/index.js`

Reply:
Ok, then keep it like is 😞